ui/src/app/routing/workspace-app-routing.tsx

```diff
@@ -142,63 +143,60 @@ export const WorkspaceRoutes = () => {
       </AppRoute>
       <AppRoute
         exact
-        path={`${path}/apps`}
-        guards={[adminLockedGuard(ns, wsid), tempAppsAnalysisGuard(ns, wsid)]}
-      >
-        <AppsListPage
-          routeData={{
-            title: 'View Apps',
-            pageKey: 'apps',
-            workspaceNavBarTab: 'apps',
-            breadcrumb: BreadcrumbType.Workspace,
-          }}
-        />
-      </AppRoute>
-      <AppRoute
-        exact
-        path={`${path}/${NOTEBOOKS_TAB_NAME}`}
+        path={`${path}/${analysisTabName}`}
         guards={[adminLockedGuard(ns, wsid)]}
       >
-        <NotebookListPage
-          routeData={{
-            title: 'View Notebooks',
-            pageKey: NOTEBOOKS_TAB_NAME,
-            workspaceNavBarTab: NOTEBOOKS_TAB_NAME,
-            breadcrumb: BreadcrumbType.Workspace,
-          }}
-        />
+        {environment.showNewAnalysisTab ? (
+          <AppsListPage
+            routeData={{
+              title: 'View Analysis Files',
```

Comment:
this title is new - what do you think?

Reply:
I like it! That seems to be appropriate for what we are looking at.